index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Car Lot</title>
+  <link rel="stylesheet" type="text/css" href="bower_components/bootstrap/dist/css/bootstrap.min.css">
+  <link rel="stylesheet" type="text/css" href="quiz.css">
+</head>
+<body>
+  <div>
+    <div class="jumbotron">
+      <h1>America's Best Car Lot.<br>Really.</h1>
+    </div>
+  </div>
+<nav class="form-group">
+  <div class="container"
+  <label>You're going to need to type stuff:</label>
+  <input type="text" id="textInputField" placeholder="type stuff in here" disabled>
+  <button id="textInputButton" class="btn btn-primary" disabled>Enter</button>
+</nav>
+
+
+<div class="container">
+  <div id="carsGoHere" class="row">
+  </div>
+</div>
+<!-- <script src="bower_components/bootstrap/dist/js/bootstrap.min.js"></script> -->
+<script type="text/javascript" src="carlot.js"></script>
+<script type="text/javascript" src="quiz_xhr.js"></script>
```

Comment:
Looking at the technical requirements, it does actually ask you to put the other modifying IFFEs into separate files, so you may need to break things up a little by the time you're done.